variables.tf

```diff
@@ -86,17 +86,29 @@ variable "layers" {
 variable "datadog_python_layer_version" {
   type        = number
   description = "The version of the Datadog Python Layer"
-  default     = 64
+  default     = 98
 }
 
 variable "datadog_extension_layer_version" {
   type        = number
   description = "The version of the Datadog Extension Layer"
-  default     = 34
+  default     = 63
 }
 
 variable "rds_logs" {
   type        = bool
   description = "Whether to create lambda resource policy for sending all /aws/rds/* cloudwatch logs to the datadog log forwarder"
   default     = true
 }
+
+variable "store_failed_events" {
+  type        = bool
+  description = "Whether to store failed events in the log forwarder"
+  default     = false
+}
+
+variable "bucket_name" {
+  description = "name of the bucket to store cache files from several functions as well as failed events"
+  type        = string
+  default     = false
```

Comment:
Maybe this should default to null or possibly ""?  We shouldn't have a boolean for a string default.